lib/tasks/check_changelog.rake

```diff
@@ -17,8 +17,13 @@
 require 'rake'
 
 namespace "check" do
-  desc "Checking for new IDs (bugzilla,fate,...)  in *.changes file"
+  desc "Checking for new IDs (bugzilla,fate,...) in *.changes file"
   task :changelog do
+    if obs_sr_project.nil?
+      puts "Skipping the check because OBS submit request project "\
```

Comment:
I'd print it on the `$stderr` and `the check` should be described - the task is called as part of another task so it might not be obvious which check was skipped.